app/contextCreator/createSolutionListPageContext.js

```diff
@@ -0,0 +1,17 @@
+export const createSolutionListPageContext = (solutions, pageTitle, pageDescription) => {
+  const context = {
+    pageTitle,
+    pageDescription,
+  };
+
+  context.solutions = solutions.map(solution => ({

```

Comment:
extract this to a separate function then in the context above we can just add solutions directly via this

Reply:
Extracted to another function.